tests/metrics/functional/test_classification.py

```diff
@@ -27,16 +25,18 @@
 from pytorch_lightning.metrics.utils import to_onehot, get_num_classes, to_categorical
 
 
-@pytest.mark.parametrize(['sklearn_metric', 'torch_metric', 'only_binary'], [
-    pytest.param(sk_accuracy, accuracy, False, id='accuracy'),
```

Comment:
This is the only line that really is changing right? (removing accuracy from tests).
If so, please remove all the other formatting changes